bin/orogen

```diff
@@ -80,7 +82,8 @@ parser = OptionParser.new do |opt|
     opt.on('--[no-]rtt-scripting', 'when generating the typekit, remove features needed for RTT scripting (speeds up compilation and reduces code size)') do |value|
         OroGen::TypekitMarshallers::TypeInfo::Plugin.rtt_scripting = value
     end
-
+    
+    opt.on("-t", "--tests")   { rtt_cpp.enable_namespace = "test"}
```

Comment:
You seem to assume that there will be only one namespace ... Should replace the '=' by a plain method call

~~~
project.enable_namespace 'test'
~~~